components/connectApiKeyExchange.tsx

```diff
@@ -64,109 +64,114 @@ const ConnectApiKeyExchange: FC<Props> = ({ exchange, onSuccess, slideOverPlace
+                                <label htmlFor="apiKey" className="block font-normal text-sm">
+                                    API Key
+                                </label>
+                                <div className="relative rounded-md shadow-sm mt-1">
+                                    <input
+                                        autoComplete="off"
+                                        placeholder="Your API Key"
+                                        autoCorrect="off"
+                                        type="text"
+                                        name="apiKey"
+                                        id="apiKey"
+                                        onChange={handleKeyChange}
+                                        className="h-12 pb-1 pt-0 focus:ring-primary focus:border-primary border-darkblue-500 block
                          placeholder:text-sm placeholder:font-normal placeholder:opacity-50 bg-darkblue-700 w-full font-semibold rounded-md placeholder-gray-400"
-                                />
+                                    />
+                                </div>
+                            </div>
+                            <div>
+                                <label htmlFor="apiSecret" className="block font-normal text-sm">
+                                    API Secret
+                                </label>
+                                <div className="relative rounded-md shadow-sm mt-1">
+                                    <input
+                                        autoComplete="off"
+                                        placeholder="Your API Secret"
+                                        autoCorrect="off"
+                                        type="text"
+                                        name="apiSecret"
+                                        id="apiSecret"
+                                        onChange={handleSecretChange}
+                                        className="h-12 pb-1 pt-0 focus:ring-primary focus:border-primary border-darkblue-500 block
+                        placeholder:text-sm placeholder:font-normal placeholder:opacity-50 bg-darkblue-700 w-full font-semibold rounded-md placeholder-gray-400"
+                                    />
+                                </div>
                             </div>
-                        </>
-                    }
-                    {
-                        ExchangeSettings.KnownSettings[exchange?.internal_name]?.AuthorizationNote &&
-                        <WarningMessage className=''>
-                            <div className='text-black'>
-                                {ExchangeSettings.KnownSettings[exchange?.internal_name]?.AuthorizationNote}
+                            {
+                                exchange?.has_keyphrase &&
+                                <>
+                                    <label htmlFor="apiKey" className="block font-normal text-sm">
```

Comment:
htmlFor is wrong, maybe because of this
<img width="1003" alt="Screen Shot 2022-11-04 at 01 23 59" src="https://user-images.githubusercontent.com/6187131/199837718-17a34036-8964-4fb5-9570-d8474e1bfec9.png">